FLEX.podspec

```diff
@@ -40,6 +40,7 @@ Pod::Spec.new do |spec|
   spec.xcconfig = {
        'CLANG_CXX_LANGUAGE_STANDARD' => 'gnu++11',
   }
+  spec.preserve_paths = '**/*.hmap',
```

Comment:
Ok I m fine with mentioning it explicit also, but even this `*.hmap` should do? So tomorrow for any reason we add more hmap's or change names we don't have to modify.

And I think *.hmap won't have that much performance downside?

Reply:
> So tomorrow for any reason we add more hmap's or change names we don't have to modify.

I feel it's a good practice to be explicit here. So that when we do start shipping new/updated hmap files, (say due to changes from upstream), then we should check if the changes are still compatible with Bazel or we need to change some rules somewhere.

But I do see your point, I guess if we start shipping more `hmap` files as part of the cocoapod, there shouldn't be any harm as such.

So I think just `*.hmap` works 👍

Reply:
Done.